pmm/pmm2-ui-tests.groovy

```diff
@@ -190,8 +190,16 @@ pipeline {
                     }
                     steps {
                         sh """
-                            mkdir testdata
-                            sudo MYSQL_IMAGE=\${MYSQL_IMAGE} MONGO_IMAGE=\${MONGO_IMAGE} POSTGRES_IMAGE=\${POSTGRES_IMAGE} PMM_SERVER_IMAGE=\${DOCKER_VERSION} docker-compose up -d
+                            docker volume create pmm-server-data
+                            docker network create -d bridge --subnet 192.168.0.0/24 --gateway 192.168.0.1 pmm-network
```

Comment:
I think it's better to declare volumes and networks inside compose stack as they won't be removed on `docker-compose --down`

Reply:
sure, I have addressed this, thanks for the input.